bin/read_correction_with_sga

```diff
@@ -15,7 +15,8 @@ Usage: read_correction_with_sga [options]
         
         -m|min_length	       <discard sequences that are shorter than this during preprocess>
         -q|quality_trim	       <trim reads using Heng Li's BWT trimming algorithm>
-        -l|quality_filter	   <discard read if it contains more than this many low-quality bases>
+        
+#        -l|quality_filter	   <discard read if it contains more than this many low-quality bases> # Not doing any filtering since 1.3.2013
```

Comment:
It's probably better just to delete all references in the one commit.  We can always revert it if you need it back later.  I can show you how you can sneak back into https://github.com/nds/assembly_improvement/commit/0f0a1304f886cda4208ac79b4e521fafa43d8235 to make the change  if you like otherwise you could just add another one to the PR